modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -758,15 +758,15 @@ public void prepareRemoval(KillEvent killEvent)
     @Override
     public void postRemoval(KillEvent ce)
     {
-        _log.info("CellAdapter : prepareRemoval : waiting for gate to open");
+        logger.info("CellAdapter : prepareRemoval : waiting for gate to open");
```

Comment:
I would suggest you continue with  string-concatenations and forget about naming issue. Leave it to the next generation :). It will keep your changes small and easy to review.